l10n_it_vat_registries_split_payment/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Lorenzo Battistini
+# Copyright (c) 2019 Matteo Bilotta
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "VAT registries + Split Payment",
+    "summary": "Bridge module to make VAT registries work with Split Payment",
```

Comment:
Come in https://github.com/OCA/l10n-italy/pull/1281#discussion_r291840238, vedi le linee guida:
https://www.odoo-italia.org/documentazione/sviluppo/guidelines.html#titolo-del-modulo

Reply:
Fatto!  
Grazie! 😉